package.json

```diff
@@ -44,6 +44,8 @@
     "@ember/optional-features": "^2.0.0",
     "@ember/test-helpers": "^2.6.0",
     "@embroider/test-setup": "^0.48.1",
+    "@glimmer/component": "^1.1.2",
+    "@glimmer/tracking": "^1.1.2",
```

Comment:
Both of these should move to `dependencies` otherwise, apps without glimmer would not be able to use this addon